### PR TITLE
chore: ID-1632 Update oidc-client-ts dependency

### PR DESCRIPTION
**Pull request by haydenfowler**
# Summary
This PR bumps `oidc-client-ts` so that we can make use of a feature that was introduced in a later version. According to the [packages releases page](https://github.com/authts/oidc-client-ts/releases?page=1), no breaking changes have been implemented.

I've tested the login & logout flow locally and they appear to be working as intended.

# Detail and impact of the change
## Changed
- Passport: Bumped `oidc-client-ts` dependency to 2.4.0

### CODE REVIEW

**Review by carmen0208 — DISMISSED** [review]

File: sdk/package.json
Comment (CodeSchwert) [inline sdk/package.json:1]: @haydenfowler can you run build locally and see if that reorders the dependencies?

Comment (haydenfowler) [reply sdk/package.json:1]: @CodeSchwert after `yarn build`, the dependencies still appear in this order. Want me to revert the re-ordering?

Comment (CodeSchwert) [reply sdk/package.json:1]: Yes, revert the ordering please.

Comment (haydenfowler) [reply sdk/package.json:1]: Fixed @CodeSchwert

**Review by CodeSchwert — APPROVED** [review]

**Review by sharif9876 — APPROVED** [review]

**Pull request merged**